dev-tools/reconfigurator-cli/tests/config.test.toml

```diff
@@ -0,0 +1 @@
+../../../nexus/tests/config.test.toml
```

Comment:
interesting! I've historically just reached out across crate boundaries, but this is undoubtedly better.